torchaudio/functional.py

```diff
@@ -823,13 +823,13 @@ def mask_along_axis_iid(specgrams, mask_param, mask_value, axis):
         specgrams (Tensor): Real spectrograms (batch, channel, freq, time)
         mask_param (int): Number of columns to be masked will be uniformly sampled from [0, mask_param]
         mask_value (float): Value to assign to the masked columns
-        axis (int): Axis to apply masking on (2 -> frequency, 3 -> time)
+        axis (int): Axis to apply masking on (-2 -> frequency, -1 -> time)
 
     Returns:
         torch.Tensor: Masked spectrograms of dimensions (batch, channel, freq, time)
     """
 
-    if axis != 2 and axis != 3:
+    if axis != -2 and axis != -1:
         raise ValueError('Only Frequency and Time masking are supported')
```

Comment:
I feel it a little bit too strict to disallow positive indices, but as long as it is explained in doctoring, I think it's fair.

Reply:
oh wait, isn't this BC breaking?